apps/etl/extraction/sources/base/handler.py

```diff
@@ -161,6 +162,7 @@ def handle_extraction(
             parent_id=parent_id,
             metadata={"input": params} if params else {},
         )
+        SentryTag.set_tags({SentryTag.Tag.SOURCE: source, SentryTag.Tag.TRACE_ID: instance.trace_id})
```

Comment:
Let's add this to the BaseExtractionV2.__init__ after the `self.extraction_object =` as well